draft-kasselman-oauth_spiffe.md

```diff
@@ -95,9 +95,9 @@ The SPIFFE deployment is responsible for bootstrapping the identifiers and crede
 
-The "register on first use" implementation varies depending whether or not the OAuth flows includes redirection. In non-redirecting flows (such as Client Credentials or Resource Owner Password Credentials), the OAuth client authenticates and requests tokens directly from the authorization server without the need for user redirection. Flows with redirection (like Authorization Code Flow) temporarily send users to an external identity provider before returning them to the application with a token or authorization code, which is then used by the OAuth client. These redirect-based flows rely on the user agent (typically a web browser) to maintain the authentication context and handle the navigation between the application and the identity provider. These fundamental differences require distinct implementation considerations when implementing client registration on first use. Redirecting flows need state management to reconnect returning users with their original session after authentication, while non-redirecting flows can create a `client_id` inline within the same request context. Properly handling both OAuth patterns ensures a smooth registration experience regardless of the authentication method selected.
+The "register on first use" implementation varies depending whether or not the OAuth flows includes redirection. In non-redirecting flows (such as Client Credentials or Resource Owner Password Credentials), the OAuth client authenticates and requests tokens directly from the authorization server without the need for user redirection. Flows with redirection (like Authorization Code Flow) temporarily send users to an external identity provider before returning them to the application with a token or authorization code, which is then used by the OAuth client. These redirect-based flows rely on the user agent (typically a web browser) to maintain the authentication context and handle the navigation between the application and the identity provider. These fundamental differences require distinct implementation considerations when implementing client registration on first use. Redirecting flows need state management to reconnect returning users with their original session after authentication, while non-redirecting flows can create a client ID inline within the same request context. Properly handling both OAuth patterns ensures a smooth registration experience regardless of the authentication method selected.
```

Comment:
Change to:

"Register on first use" varies depending whether or not the OAuth flows includes redirection. 

Remove "fundamental" from below sentence

These fundamental differences require distinct implementation considerations when implementing client registration on first use.